Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardContent = () => {
+	return (
+       <div className="card-content">
+          <h2>Get Started With React</h2>
+          <p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>
+           <a href="https://www.reactjs.org" className="react-link">reactjs.org</a>
```

Comment:
After you get a little more practice with `state` you should come back and see if you can make this component reusable. Keep simplicity in mind when you build this out. The more complicated you go the harder it will be to debug.